app/views/navigation.html

```diff
@@ -3,16 +3,15 @@
     <div class="branding">
       <div class="nav navbar-nav navbar-right">
         <span class="w3c"><a href="http://w3.org/">
-          <img alt="W3C" width="85" height="50" src="images/w3c.png">
+          <img alt="W3C" width="110" height="75" src="images/w3c.png">
```

Comment:
Undo height changes please, this breaks current design, or is there a reason it needs to be this size?
(And by all means keep the right aspect ratio for the logo)